rti_connext_dds_cmake_module/cmake/rti_build_helper.cmake

```diff
@@ -516,11 +517,7 @@ function(rti_guess_connextdds_arch)
           endif()
         endif()
         if(NOT CONNEXTDDS_ARCH)
-          message(STATUS
-            "unsupported CMAKE_HOST_SYSTEM_NAME (${CMAKE_HOST_SYSTEM_NAME}) "
-            "or CMAKE_HOST_SYSTEM_PROCESSOR (${CMAKE_HOST_SYSTEM_PROCESSOR}). "
-            "Using architecture ${architecture_name} anyway.")
-          set(CONNEXTDDS_ARCH "${architecture_name}")
+          list(APPEND architecture_candidates ${architecture_name})
```

Comment:
i think they are not candidates any more, they are `unsupported_architecture` (the only difference is that does not have `java` in the list.) runtime library list for the host system?

Reply:
yeah i think they should be just unsopported too, i did this since the original code would set this unsopported architecture anyway, so i called them candidates...